Makefile.common

```diff
@@ -51,6 +52,31 @@ cacheartifact:
 	cekit-cache add ${ABSOLUTE_LOCAL_ARTIFACT_DIR} --md5 ${MD5SUM}
 	sed -i 's/- md5:.*/- md5: ${MD5SUM}/' modules/install-artifact/module.yaml 
 
+pushfrombrew:
+	$(eval SUBDIR_START_LINE=$(shell grep -n -m 1 $(CURDIR) $(CEKIT_OUTPUT_FILE) | cut -f1 -d:))
```

Comment:
There's quite a lot going on in this target, I wonder if separating the logic into a separate shellscript that is invoked from here would be more maintainable?

Reply:
The syntax is easier in a separate script.  Makefile syntax adds a lot of constraints.  And makes the script longer, by forcing extra logic into the loop.

I felt there were too many variables being passed from the Makefile to the script.  5 from the Makefile.
I'll take a 2nd look, I can combine 3 into 1 (which would leave only 3 args):  NEW_VAR=$(DOCKER_REGISTRY)/$(REPO):$(IMAGE_VERSION)

How many variables do you feel are acceptable to pass?